eng/tools/ToolMetadataExporter/Utility.cs

```diff
@@ -0,0 +1,281 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Text.Json;
+using ToolSelection.Models;
+
+namespace ToolMetadataExporter;
+
+internal class Utility
```

Comment:
Taken from ToolDescriptionEvaluator and modified. Need to update logic together.